assets/js/dashboard/stats/graph/top-stats.js

```diff
@@ -64,6 +64,13 @@ export default function TopStats({ data, onMetricUpdate, tooltipBoundary }) {
             <SecondsSinceLastLoad lastLoadTimestamp={lastLoadTimestamp} />s ago
           </p>
         )}
+
+        {stat.name === 'Scroll depth' &&
+          stat.warning_code === 'no_imported_scroll_depth' && (
+            <p className="font-normal text-xs whitespace-nowrap">
+              * Includes only native data
```

Comment:
```suggestion
              * Does not include imported data
```

I don't think "native" terminology is not mentioned anywhere in the app and would be confusing.

Reply:
Good point. Changed now!